S7.Net.UnitTest/S7NetTestsAsync.cs

```diff
@@ -815,6 +839,19 @@ public partial class S7NetTests
             Assert.AreEqual(test_value, test_value2, 0.01, "Compare Write/Read"); //Need delta here because S7 only has 32 bit reals
         }
 
+        [TestMethod]
+        public async Task Test_Async_ReadWriteSingle()
+        {
+            float test_value = 55.6632f;
+            await plc.WriteAsync("DB1.DBD0", test_value);
+            Assert.AreEqual(plc.LastErrorCode, ErrorCode.NoError, "Write Single");
```

Comment:
The third parameter for `Assert` should be a message (i.e. "After WriteAsync Plc.LastErrorCode should be 'NoError', but LastErrorCode is '{plc.ErrorCode}'."). I guess this isn't applied anywhere though ;-)

Reply:
Yeah, UnitTests needs a review, I may start on this.